modules/light-clients/08-wasm/types/upgrade.go

```diff
@@ -45,8 +45,8 @@ func (cs ClientState) VerifyUpgradeAndUpdateState(
 
 	payload := sudoMsg{
 		VerifyUpgradeAndUpdateState: &verifyUpgradeAndUpdateStateMsg{
-			UpgradeClientState:         upgradedClient,
-			UpgradeConsensusState:      upgradedConsState,
+			UpgradeClientState:         *wasmUpgradeClientState,
+			UpgradeConsensusState:      *wasmUpgradeConsState,
```

Comment:
This will panic if `wasmUpgradeClientState` or `wasmUpgradeConsState` is nil. Is it possible to assert this earlier and possibly handle it as an error there?

Not a big deal though as it's probably ok for it to panic. Or it is somehow not possible for it to reach here if it were `nil`.

Reply:
`nil` is handled by type assertion. Type assertions yield false if the interface you're trying to cast from is `nil` or `!= T` (target type).